source/manual/ab-testing.html.md

```diff
@@ -2,7 +2,7 @@
 title: Run an A/B or multivariate test
 parent: "/manual.html"
 layout: manual_layout
-section: Tools
+section: Development & Reviews
```

Comment:
I'm not sure this really fits in "Development & Reviews", but it doesn't fit in "Tools" either so I'm not sure what to suggest.

Reply:
I've given it it's very own category, we should split up the page at some point anyway.